io.github.pieterdd.StretchBreak.yaml

```diff
@@ -0,0 +1,45 @@
+id: io.github.pieterdd.StretchBreak
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: stretch-break
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--device=dri"
+- "--env=G_MESSAGES_DEBUG=none"
+- "--env=RUST_BACKTRACE=1"
+- "--own-name=io.pieterdd.StretchBreak.Core"
```

Comment:
you should use upstream a name that matches the app-id. D-Bus as similar namespacing requirements where one doesn't simply take a random name. Then there is no need to `own-name`